src/controller.cpp

```diff
@@ -54,10 +54,7 @@ extern "C" uint32_t read_controller() {
     }
 
     Cheats::apply_cheats();
-    if (mm_visible || prac_visible || settings_visible || cheats_visible ||
-        tools_visible || inventory_visible || item_wheel_visible || pause_visible ||
-        memory_visible || warping_visible || flags_menu_visible ||
-        scene_menu_visible || any_saves_visible || hundo_saves_visible) {
+    if (MenuRendering::is_menu_open() == true) {
```

Comment:
This is just my opinion, but the "== true" is unnecessary and should be removed. I think here specifically it would make it easier to read. Don't think there is ever a reason to compare a boolean against a boolean literal, but that is obviously just an opinion.